enterprise/management/commands/backfill_ecu_table_user_foreign_key.py

```diff
@@ -0,0 +1,132 @@
+"""
+Management command for assigning enterprise_learner roles
+to existing linked enterprise users that are missing them.
+"""
+
+import logging
+from time import sleep
+
+from django.contrib import auth
+from django.core.management.base import BaseCommand
+from django.db import DatabaseError, transaction
+from django.db.models import Q
+
+from enterprise.models import EnterpriseCustomerUser
+from integrated_channels.utils import batch_by_pk
+
+log = logging.getLogger(__name__)
+User = auth.get_user_model()
+
+
+def safe_bulk_update(entries, properties, max_retries):
+    """Performs bulk_update with retry logic."""
+    for attempt in range(1, max_retries + 1):
+        try:
+            with transaction.atomic():
+                EnterpriseCustomerUser.objects.bulk_update(
```

Comment:
This also needs to be `all_objects` (I just noticed this now from testing out with SQL logging enabled).